utils/config/daos_server.yml

```diff
@@ -70,12 +70,13 @@
 #
 ## Network provider
 #
-## Force a specific libfabric provider to be used by all the engines.
-## The default provider depends on the interfaces that will be auto-detected:
-##  ofi+verbs for Infiniband/RoCE and
-##  ofi+tcp for non-RDMA-capable Ethernet.
+## Set the libfabric provider to be used by all the engines.
+## There is no default - run "daos_server network scan" to list the
+## providers that are supported on the fabric interfaces. Examples:
+##  ofi+verbs;ofi_rxm  for Infiniband/RoCE and
+##  ofi+tcp            for non-RDMA-capable fabrics
```

Comment:
From what I understand
```suggestion
##  ofi+tcp;ofi_rxm            for non-RDMA-capable fabrics
```

Reply:
that's not my understanding ... see "daos_server network scan -p all"

Reply:
ofi_rxm is automatically added when ofi+tcp or ofi+verbs is specified as a provider. That's true for 2.2 and beyond.

Reply:
i see, the network scan output of 2.0 and 2.2 differs in this respect: v2.0 only reports `ofi+tcp`, whereas 2.2 reports both `ofi+tcp` and `ofi+tcp;ofi_rxm`.
will make this consistent by explicitly listing `;ofi_rxm` for verbs and tcp for all releases...